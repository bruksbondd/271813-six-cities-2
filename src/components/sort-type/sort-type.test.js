import React from 'react';
import renderer from 'react-test-renderer';
import SortType from "./sort-type.jsx";

it(`render correctly sort type`, () => {
  const sortType = renderer
    .create(<SortType
      handlerSortButton={() => {}}
      typeSorting={`Popular`}
    />)
    .toJSON();
  expect(sortType).toMatchSnapshot();
});
