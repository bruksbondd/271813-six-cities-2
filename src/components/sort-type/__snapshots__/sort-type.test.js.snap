// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly sort type 1`] = `
<span
  className="places__sorting-type"
  onClick={[Function]}
  tabIndex="0"
>
  Popular
  <svg
    className="places__sorting-arrow"
    height="4"
    width="7"
  >
    <use
      xlinkHref="#icon-arrow-select"
    />
  </svg>
</span>
`;
